### PR TITLE
Fix borked busts

### PR DESCRIPTION
**Pull request by LamseyMcBrien**
Fixes for Dheria, Mithyn, Nalili, Wendis, Xestris, and especially Grynyth whose portrait has been every bit as much of a pain in my ass as the character herself has been in Simon's.

Subsequent commits: Qum the Grey, Megail the Gold, and Chosen the Explosive.

### CODE REVIEW

File: Bust Config.rb [hunk 1]
```diff
@@ -44,6 +44,88 @@
     face_name: "Z Andra emo",
     face_indexes: [4],
   },
+  # Grynyth needs separate images for every face because her semi-transparent
+  # elements and moving eyebrow thingies make overlays a total nightmare
```
Comment (Rachnera) [inline Bust Config.rb]: Actual hell.

File: Bust Config.rb [hunk 2]
```diff
@@ -1076,6 +1165,44 @@ def never
...
+Busty::BASE_CONFIG["Grynyth2-3"] = Busty::BASE_CONFIG["Grynyth"]
+Busty::MESSAGE_CONFIG["Grynyth2-3"] = Busty::MESSAGE_CONFIG["Grynyth"]
+Busty::BASE_CONFIG["Grynyth2-4"] = Busty::BASE_CONFIG["Grynyth"]
+Busty::MESSAGE_CONFIG["Grynyth2-4"] = Busty::MESSAGE_CONFIG["Grynyth"]
+Busty::BASE_CONFIG["Grynyth2-5"] = Busty::BASE_CONFIG["Grynyth"]
+Busty::MESSAGE_CONFIG["Grynyth2-5"] = Busty::MESSAGE_CONFIG["Grynyth"]
+Busty::BASE_CONFIG["Grynyth2-6"] = Busty::BASE_CONFIG["Grynyth"]
+Busty::MESSAGE_CONFIG["Grynyth2-6"] = Busty::MESSAGE_CONFIG["Grynyth"]
+Busty::BASE_CONFIG["Grynyth2-7"] = Busty::BASE_CONFIG["Grynyth"]
+Busty::MESSAGE_CONFIG["Grynyth2-7"] = Busty::MESSAGE_CONFIG["Grynyth"]
```
Comment (Rachnera) [inline Bust Config.rb]: Let's have mercy of whoever will have to add Grynyth's third faceset in the future:
```rb
(1..2).each do |i|
  (0..7).each do |j|
    name = "Grynyth#{i}-{j}"
    Busty::SUBSET_TO_BUST.insert(-1, { character_name: name, face_name: "Grynyth emo#{i > 1 ? i : ''}", face_indexes: [j] })
    Busty::BASE_CONFIG[name] = Busty::BASE_CONFIG["Grynyth"]
    Busty::MESSAGE_CONFIG[name] = Busty::MESSAGE_CONFIG["Grynyth"]
  end
end
```
(I didn't actually run the snippet, so there's a 200% chance it breaks on a typo or something)

**Review by Rachnera — COMMENTED** [review]
Poor Mithyn. Your greatness was too much for the world.

File: Bust Config.rb (hunk 2 above)
Comment (LamseyMcBrien) [reply Bust Config.rb]: Done. Your prediction was correct, but only one missing `#`, so not bad.

**Comment by LamseyMcBrien** [comment]
That's all the broken/missing ones on my list. Still need to work through my list of busts with cutoffs, but that's low-priority and can wait for another day. I'll assume I don't need to go hunting for the places where full-size busts are displayed unless I hear otherwise.

File: Bust Config.rb [hunk 3]
```diff
@@ -553,6 +563,12 @@ def never
     face_offset_x: 62,
     face_offset_y: 40,
   },
+  "Qum D'umpe2" => {
+    bust_scale: 0.75,
+    face_offset_x: 62,
+    face_offset_y: 40,
+    face_border_width: [0, 0, 0, 12],
```
Comment (Rachnera) [inline Bust Config.rb:564]: To make the file (slightly) easier to maintain (i.e. move all versions of Qum at once by changing a single bust_offest_x/y), might be better to use the supermode loop below and just add under it:
```rb
Busty::BASE_CONFIG["Qum D'umpe2"][:face_border_width] = [0, 0, 0, 12];
```

**Comment by Rachnera** [comment]
> I'll assume I don't need to go hunting for the places where full-size busts are displayed unless I hear otherwise.

Yeah, we should be good with the current changes. Worst case, busts aren't displayed in a place where they should be, but it's less of a problem in that direction.

File: Bust Config.rb (hunk 3 above)
Comment (LamseyMcBrien) [reply Bust Config.rb:564]: Done - the border width is harmless for her main bust, so I've just applied it to both.

**Pull request merged**